php/commands/user.php

```diff
@@ -739,6 +739,14 @@ public function import_csv( $args, $assoc_args ) {
 				$user->add_role( $secondary_role );
 			}
 
+			//User meta: meta_key_{meta_key}
+			foreach( $new_user as $column => $meta_value ) {
+				if ( preg_match( '/^meta_key_([A-Za-z0-9_]+)$/', $column, $matches ) ) {
+					$meta_key = $matches[1];
+					add_user_meta( $user_id, $meta_key, $meta_value );
```

Comment:
Yeah, this is something I had thought of. I wasn't sure how best to approach it. With existing user & meta key you can do one of three things

1. Add an additional value (`add_user_meta`)
2. Update all existing values to this new value
3. Update a specific meta value to this new value

You can distinguish between (1) and (2) by expecting `add_meta_key_{meta-key}` and `update_meta_key_{meta-key}` - though I'm not a fan. But trying to support (3) would be even messier.